Automattic/jetpack
# Admin Menu: Replace "Edit CSS" with "Additional CSS"

Fixes Automattic/wp-calypso#51386

#### Changes proposed in this Pull Request:

* Replaces the "Edit CSS" label with "Additional CSS" since that's the language used in the Customizer, and it's designed to add CSS rather than edit the theme's CSS

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






I don't think that I'm actually able to test this change because I'm unable to sandbox a WordPress.com site, but it should be something like this:

* Apply this change for a WP.com simple site 
* Make an API request to `/sites/ID/admin-menu`
* Check the children under the "Appearance" menu 
* Verify the label says "Additional CSS"

<img width="975" alt="Screenshot 2021-04-01 at 14 43 41" src="https://user-images.githubusercontent.com/43215253/113304741-3bf01080-92fa-11eb-8722-21ca3e4d516b.png">
